code/game/turfs/open/water.dm

```diff
@@ -8,30 +8,29 @@
 	planetary_atmos = TRUE
 	slowdown = 1
 	bullet_sizzle = TRUE
-	bullet_bounce_sound = null //needs a splashing sound one day.
+	bullet_bounce_sound = null //needs a splashing sound one day. //ShizCalev, 5 years ago.
 
 	footstep = FOOTSTEP_WATER
 	barefootstep = FOOTSTEP_WATER
 	clawfootstep = FOOTSTEP_WATER
 	heavyfootstep = FOOTSTEP_WATER
 
-	var/datum/reagent/reagent_to_extract = /datum/reagent/water
+	var/datum/reagent/reagent_to_extract = /datum/reagent/water/dirty
 	var/extracted_reagent_visible_name = "water"
 
-/*
+
 /turf/open/water/attackby(obj/item/tool, mob/user, params)
```

Comment:
I remember why it was commented out. You have two attackbys declared, one for fish, one for not fish. There should only be 1 attackby. Take a look at another object that has multiple interactions on attack for an example